applications/tari_base_node/windows/runtime/run_the_base_node.bat

```diff
@@ -45,6 +45,8 @@
     ) else (
         @echo.
         @echo Note: "%sqlite_runtime%" not found in the default location or in the system path; this may be a problem
+        @echo.
+        @pause
```

Comment:
I am not sure this pause is necessary

Reply:
It is good to let the user know about the potential problem, with a pause, otherwise, if there is a runtime problem due to SQLite out available, it is less friendly.